web/src/views/Nav.vue

```diff
@@ -0,0 +1,30 @@
+<template>
```

Comment:
This looks like a duplicate of Toolbar.vue. Are you planning on merging them together?